### ISSUE TITLE
Pagination attributes being ignored

### ISSUE THREAD

**Issue by fred** [omise/omise-ruby#4]
When querying the list resources, the pagination attributes are not taken into account.

for example, when issuing this list api call:

``` ruby
Omise::Charge.list(from: "2015-04-01", to: "2015-05-01", limit: 2, offset: 1)
```

Response gives out 20 results, ignore the attributes and date filtering.

``` ruby
<Omise::List:0x007f810c1b9470 
@attributes={
"object"=>"list", 
"from"=>"1970-01-01T00:00:00+00:00", 
"to"=>"2015-05-05T06:09:41+00:00", 
"offset"=>0, 
"limit"=>20, 
"total"=>36,
"data"=>[....]
```

**Comment by robinclart** [comment]
The arguments aren't forwarded in `Omise::Resource#get`. I already fixed it in local with a few other bugfixes and the refunds.

**Comment by robinclart** [comment]
Fixed in ce32889
